plug.vim

```diff
@@ -352,18 +357,14 @@ if s:is_win
   endfunction
 
   " Copied from fzf
```

Comment:
@janlazo Should I backport this to fzf?

Reply:
Sure, after it's merged. I haven't tested the code myself to verify that it's enough.